### DISCUSSION TITLE
Multiple Entry Card with Thresholds

### DISCUSSION THREAD

**Discussion by rspierenburg** [kalkih/mini-graph-card#1098] [Q&A] [answered]
Hi.
I'm trying to setup a Humidity sensor that shows a value with color thresholds included.  I want to overlay a weather humidity sensor as a thin red line.  Is it possible to integrate color thresholds and a secondary entity on the same card?

**Comment by ildar170975** [comment, marked as answer]
1. You can have multiple graphs on one card. See examples in Docs.
2. Currently the `color_thresholds` option is global for ALL graphs inside a card. There is a registered FR to make it per-entity.
